config/crd/patches/webhook_in_kogitoserverlessworkflows.yaml

```diff
@@ -1,16 +0,0 @@
-# The following patch enables a conversion webhook for the CRD
```

Comment:
isn't better to keep it? it will be re-added if we decide to merge the webhooks validations.

Reply:
ah, nevermind, it is added below.